shell/common/rasterizer.cc

```diff
@@ -802,28 +810,143 @@ static sk_sp<SkData> ScreenshotLayerTreeAsPicture(
   return recorder.finishRecordingAsPicture()->serialize(&procs);
 }
 
-sk_sp<SkData> Rasterizer::ScreenshotLayerTreeAsImage(
-    flutter::LayerTree* tree,
+static void RenderFrameForScreenshot(
     flutter::CompositorContext& compositor_context,
+    DlCanvas* canvas,
+    flutter::LayerTree* tree,
     GrDirectContext* surface_context,
+    const std::shared_ptr<impeller::AiksContext>& aiks_context) {
+  // There is no root surface transformation for the screenshot layer. Reset
+  // the matrix to identity.
+  SkMatrix root_surface_transformation;
+  root_surface_transformation.reset();
+
+  auto frame = compositor_context.AcquireFrame(
+      surface_context,              // skia context
```

Comment:
The style for these should be `/*arg_name=*/` so that they are checked by the linter.

Reply:
Done in follow up.